File: Solutions/JunkyDragon/Game Shop/Part2/Armor.java
package com.github.junkydragon;

/**
 * Created by Administrator on 2017-05-23.
 */
public class Armor extends Item{
    private int defence;
    public Armor(){super();}
    public Armor(String name, String description, float weight, int value, int defence) {
        super(name, description, weight, value);
        this.defence = defence;
    }
    public void Describe() {
        super.Describe();
        System.out.println("Defence\t\t\t = " +this.defence);
    }
}

File: Solutions/JunkyDragon/Game Shop/Part2/Armor.kt
package org.junkydragon.item

/**
 * Created by Administrator on 2017-05-24.
 */

class Armor(name: String, description: String, weight: Double, value: Int, val defence: Int) : Item(name, description, weight, value){

    override fun About() {
        super.About()
        println("Defence\t\t\t= $defence")
    }
}